ui/jobs/jobs.js

```diff
@@ -1563,6 +1564,95 @@ class Bars {
     };
   }
 
+  setupSch() {
+    let gcd = kUnknownGCD;
+
+    let aetherflowStackBox = this.addResourceBox({
+      classList: ['sch-color-aetherflow'],
+    });
+
+    let fairyGaugeBox = this.addResourceBox({
+      classList: ['sch-color-fairygauge'],
+    });
+
+    let bioBox = this.addProcBox({
+      id: 'sch-procs-bio',
+      fgColor: 'sch-color-bio',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    let aetherflowBox = this.addProcBox({
+      id: 'sch-procs-aetherflow',
+      fgColor: 'sch-color-aetherflow',
+      scale: gcd,
+      threshold: gcd * 3,
+    });
+
+    let lucidBox = this.addProcBox({
+      id: 'sch-procs-luciddreaming',
+      fgColor: 'sch-color-lucid',
+      scale: gcd,
+      threshold: gcd + 1,
+    });
+
+    this.jobFuncs.push((jobDetail) => {
+      let aetherflow = jobDetail.aetherflowStacks;
+      let fairygauge = jobDetail.fairyGauge;
+      let milli = (jobDetail.fairyMilliseconds / 1000).toFixed(0);
+      aetherflowStackBox.innerText = aetherflow;
+      fairyGaugeBox.innerText = fairygauge;
+      let f = fairyGaugeBox.parentNode;
+      if (milli != 0) {
+        f.classList.add('bright');
+        fairyGaugeBox.innerText = milli;
+      } else {
+        f.classList.remove('bright');
+        fairyGaugeBox.innerText = fairygauge;
+      }
+
+      aetherflowBox.threshold = gcd * aetherflow;
+
+      let p = aetherflowStackBox.parentNode;
+      let s = parseFloat(aetherflowBox.duration) - parseFloat(aetherflowBox.elapsed);
```

Comment:
duration can be `null` here sometimes, which makes it possible for `s` to be `NaN`.  I think this might be more accurately written as:

`const s = parseFloat(aetherflowBox.duration || 0) - parseFloat(aetherflowBox.elapsed);`

...and this will make the box turn red when aetherflow is off cooldown (regardless of stacks).  I think that's a good thing?

Reply:
Sounds goodヽ(^o^)丿